templates/element/menu/nav-item.php

```diff
@@ -0,0 +1,18 @@
+<?php
+$options = $item->getOptions();
+$name = $name ?? $this->getName();
+
+$classes[] = $options['class'] ?? null;
+$classes[] = 'nav-link';
```

Comment:
`nav-item.php` and `dropdown-item.php` seem identical expect for couple of class name changes. It's better to keep the old filename `item.php` and just parameterize the class names which need to change.